geowebcache/core/src/test/java/org/geowebcache/storage/AbstractBlobStoreTest.java

```diff
@@ -959,7 +962,9 @@ public void testDeleteByParametersIdDoesNotDeleteOthers() throws Exception {
         store.put(toCache1);
         store.put(toCache2);
         store.deleteByParametersId("testLayer", paramID1);
-        assertThat(store.get(fromCache2_3), is(true));
+
+        await().atMost(5, SECONDS) // give stores with async deletes a chance to complete
```

Comment:
Github actions are often very unpredictable, time-wise. I would give it a much larger max wait... if I remember correctly, this is the sort of wait that ends the moment the condition is met, and the condition is actually tested much more often.

Reply:
changed it to 30s